src/lib/mongoBackend/mongoUnsubscribeContext.cpp

```diff
@@ -111,13 +111,23 @@ HttpStatusCode mongoUnsubscribeContext(UnsubscribeContextRequest* requestP, Unsu
     /* Destroy any previous ONTIMEINTERVAL thread */
     getNotifier()->destroyOntimeIntervalThreads(requestP->subscriptionId.get());
 
-    responseP->statusCode.fill(SccOk);
-    reqSemGive(__FUNCTION__, "ngsi10 unsubscribe request", reqSemTaken);
+
+    // FIXME P7: mongoSubCache stuff should be avoided if subscription is not patterned
```

Comment:
It's the same thing as before, the call to mongoSubCacheItemLookup could be avoided, if we know that the sub is not patterned.